### DISCUSSION TITLE
How to finally get rid of black triangles in themes for oh-my-posh in latest WT Preview?

### DISCUSSION THREAD

**Discussion by carpet92** [microsoft/terminal#13553] [Q&A]
If I got it correctly then there is info about fix the https://github.com/microsoft/terminal/issues/7014 bug in changelog for [1.15.1862.0](https://github.com/microsoft/terminal/releases/tag/v1.15.1862.0):

---
- We no longer suppress black blackground or gray foreground for PowerShell (https://github.com/microsoft/terminal/pull/13352)
    - We have chosen to remove this workaround as newer versions of PowerShell's PSReadline component contain a fix for the issue.
    - This was a compatibility band-aid that was impacting the capabilities of great projects such as [Oh My Posh](https://ohmyposh.dev/).
    - ❗ If you see unexpected black backgrounds appearing behind text while typing a command in PowerShell, make sure your PSReadline version is up to date. You can update your version of PSReadline by running the command, Update-Module PSReadline.
---

Well, I have installed PSReadLine 2.2.6 by `Install-Module -Name PSReadline -Force` for PowerShell 7.2.5. But for me the issue is still reproduces on the latest 1.15.1862.0.

Related discussion in `oh-my-posh` repository: https://github.com/JanDeDobbeleer/oh-my-posh/discussions/2555

Maybe I have to do something on my end in WT config to fix that behavior? How to fix the issue as it looks done in this https://github.com/microsoft/terminal/issues/7014#issuecomment-663464319 comment?

---

Setup:

```
Windows Terminal Preview 1.15.1862.0 
PowerShell 7.2.5
PSReadLine 2.2.6
oh-my-posh 8.20.1
Hack Nerd Font (same happens with CaskaydiaCove Nerd Font)

Terminal-Icons 0.10.0
```

$PROFILE:

```
clear
oh-my-posh init pwsh --config "$env:POSH_THEMES_PATH\agnoster.omp.json" | Invoke-Expression
Import-Module Terminal-Icons
New-Alias code code-insiders
```

settings.json - https://pastebin.com/raw/pUghg5QZ

---

Edit: it surely fixed the https://github.com/microsoft/terminal/issues/13513 but not black triangles/themes.

**Comment by DHowett** [comment]
So right now, we don't have a good fix for issue #7014, but it's coming up pretty soon as something the new text rendering engine should handle. I'm pretty excited, so watch this space. Sorry we don't have a better answer right now.

**Reply by divado** [reply]
Don't want to be pushy, but are there any updates on the issue?

**Reply by zadjii-msft** [reply]
There haven't been any updates in that thread, so there hasn't been any progress.
